docs: update browser compatibility (#1705)
siemens/ix: packages/documentation/docs/faq.md

</Accordion>

<Accordion title="Which browsers are supported?" id="vendor-support">
Our design system is compatible with most modern browsers, ensuring broad accessibility and functionality across platforms. 
Although we aim to provide a smooth experience across all browsers, we recommend using **Google Chrome** for optimal performance and full feature support. 
  
In addition to Chrome, the design system is also compatible with other popular browsers such as Microsoft Edge, Firefox, and Safari.

> However, users of older or less common browsers may experience limited functionality, as our system could rely on features that are not supported or have partial support in certain versions.

Reviewer: ```suggestion
However, users of older or less common browsers may experience limited functionality, as our system could rely on features that are not supported or only have partial support in certain versions.
```

> The table below outlines the supported versions for different browsers. 

Reviewer: ```suggestion
The table below lists the supported versions for various browsers.
```

<table class="table" >
  <thead>
    <tr>

>       <th scope="col">Version</th>

Reviewer: ```suggestion
      <th scope="col">iX version</th>
```

Author: Should be alligned with "release-info" which uses "Version"

      <th scope="col">Chrome</th>
      <th scope="col">Edge</th>
      <th scope="col">Firefox</th>
      <th scope="col">Safari</th>
      <th scope="col">Internet Explorer</th>
      <th scope="col">Pre-Chromium Edge</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">v2.0.0</th>
      <td>✅ v122+</td>
      <td>✅ v122+</td>
      <td>✅ v122+</td>

>       <td>✅ v17+</td>

Reviewer: Why exactly 17? The other browsers where all released around February 2024, while this version is from 2023.
Is there a reason behind that?

Author: The general idea was to use full versions that is maximum 1 year old, we can also use 17.3, which have beem released in January 2024, if it suits better.

      <td>❌</td>
      <td>❌</td>
    </tr>
  </tbody>
</table>

> Please be aware that while our testing primarily focuses on Chrome, we may not cover every browser. 

Reviewer: What are we trying to say here?

```suggestion
Please note that our testing primarily focuses on Chrome, so coverage for other browser vendors may not be as detailed.
```
